inventory_management_system_api/services/utils.py

```diff
@@ -116,24 +116,33 @@ def _validate_catalogue_item_property_value(defined_property: Dict, supplied_pro
 
     defined_type = defined_property["type"]
     defined_allowed_values = defined_property["allowed_values"]
+    defined_value_mandatory = defined_property["mandatory"]
```

Comment:
I am not sure about the usage of `value` in this one, maybe `defined_property_mandatory`? The two above may need to change as well to keep things consistent `defined_property_type` etc.